Build fails

After `opam pin add functoria . ; opam install -d functoria`.

```
ocaml setup.ml -build
Finished, 1 target (0 cached) in 00:00:00.
+ /Users/mort/.opam/system/bin/ocamlfind ocamlc -c -g -annot -bin-annot -safe-string -short-paths -keep-locs -strict-sequence -I runtime -package cmdliner -package dynlink -package fmt -package fmt.tty -package ocamlgraph -package rresult -package unix -warn-error +1..49 -w +A-4-6-7-9-40-42-44-48 -I src -I runtime -o src/functoria_key.cmo src/functoria_key.ml
File "src/functoria_key.ml", line 119, characters 24-35:
Error: Unbound value Arg.env_var
Command exited with code 2.
Compilation unsuccessful after building 36 targets (0 cached) in 00:00:01.
E: Failure("Command ''/usr/local/bin/ocamlbuild' runtime/functoria-runtime.cma runtime/functoria-runtime.cmxa runtime/functoria-runtime.a runtime/functoria-runtime.cmxs src/functoria.cma src/functoria.cmxa src/functoria.a src/functoria.cmxs -tag debug' terminated with error code 10")
gnumake: *** [build] Error 1
```

--- a/opam
+++ b/opam
@@ -20,7 +20,7 @@ remove: ["ocamlfind" "remove" "functoria"]
 depends: [
   "ocamlfind" {build}
   "base-unix"
-  "cmdliner"
+  "cmdliner" {>= "0.9.8"}
   "rresult"
   "fmt"
   "ocamlgraph"

